pkg/cloudprovider/providers/aws/aws.go

```diff
@@ -68,6 +68,7 @@ const TagNameSubnetPublicELB = "kubernetes.io/role/elb"
 // Currently we accept only the value "0.0.0.0/0" - other values are an error.
 // This lets us define more advanced semantics in future.
 const ServiceAnnotationLoadBalancerInternal = "service.beta.kubernetes.io/aws-load-balancer-internal"
+const ServiceAnnotationLoadBalancerName = "service.beta.kubernetes.io/aws-load-balancer-name"
```

Comment:
needs a comment, especially that this is a full name, and not a prefix.

@justinsb are you ok with a full name here?